tools/mtmd/clip.cpp

```diff
@@ -1655,6 +1667,17 @@ struct clip_graph {
             inpL = cur;
         }
 
+        // TODO @ngxson : find a way to move this outside
+        if (ctx->proj_type == PROJECTOR_TYPE_QWEN2A) {
+            ggml_tensor * cur = inpL;
+            cur = ggml_transpose(ctx0, cur);
+            cur = ggml_cast(ctx0, cur, GGML_TYPE_F32);
```

Comment:
Any reason to prefer `ggml_cast` here over `ggml_cont`?

Reply:
I had some problem with `ggml_compute_forward_pool_1d` and when looking into the source code, I mistakenly thought that it only supports F16. Changed to `ggml_cont` in https://github.com/ggml-org/llama.cpp/pull/13760/commits/e53a0dc43f0869388e6169547abfc112184ab84d